sources/README.md

```diff
@@ -142,6 +142,20 @@ _Neighborhoods within the City of Atlanta._
 * license: _https://www.arcgis.com/home/item.html?id=716f417a1990446389ef7fd2c381d09f_
 * usage: `property`, `geometry`
 
+## Collaborative Australian Protected Areas Database
+
+_The Collaborative Australian Protected Areas Database (CAPAD) provides both spatial and textual information about government, Indigenous and privately protected areas for Australia, in both the marine and terrestrial environments._ 
+
+* added: `2021-09-02`
+* id: `1746286501`
+* name: `au-capad`
+* prefix: `au-capad`
+* license_type: _CC BY (equivalent)_
+* license_text: _You are free to: Share — copy and redistribute the material in any medium or format, Adapt — remix, transform, and build upon the material for any purpose, even commercially._
```

Comment:
Just like the others... prefix with `Collaborative Australian Protected Areas Database (CAPAD) 2020, Commonwealth of Australia 2021`

Reply:
I added this snippet to the source json in a follow up commit.. should be good here.